config/s3/test_multipart_upload.yaml

```diff
@@ -342,7 +342,7 @@ test_32702_2:
           count: 1
 
 test_40265:
-  file_size: 6
+  file_size: 10
```

Comment:
What is difference between file_size and chunk_size?

Reply:
file size, is size of file to be created. chunk size is used to read the data in that size of chunks in created file and make parts out of it 
  data = file_pointer.read(int(part_size * chunk_size))

Reply:
@archanabpatil I didn't get about 6, is it in MB, KB or something else?